ext/storage_sources/azure_store/CMakeLists.txt

```diff
@@ -27,12 +27,12 @@ target_compile_options(
 target_link_libraries(
     azure_container_util
     PUBLIC
-        azure_core_lib
-        azure_storage_lib
-        azure_storage_common_lib
+        Azure::azure-core
+        Azure::azure-storage-blobs
+        Azure::azure-storage-common
 )
 
-add_dependencies(azure_container_util azure-sdk)
+#add_dependencies(azure_container_util azure-sdk)
```

Comment:
Why was this commented out?

Reply:
[3433d6a](https://github.com/wiredtiger/wiredtiger/pull/9000/commits/3433d6a7002b208627e313092587c0b6cf019f25)

Reply:
I was meant to remove this, the add dependencies is added within the azure_sdk.cmake file instead!